share/spice/symbolab/symbolab.js

```diff
@@ -0,0 +1,48 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_symbolab = function (api_result) {
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || !api_result.solution) {
+            return Spice.failed('symbolab');
+        }
+
+        DDG.require('mathquill', function (){
+            Spice.add({
+                id: "symbolab",
+                name: "Calculator",
+                data: api_result.solution,
+                meta: {
+                    sourceName: "Symbolab.com",
+                    sourceUrl: api_result.solution.url
+                },
+                normalize: function (data) {
+                    return {
+                        latex: data.originalQuery.replace(/ /g, "\\text{ }"),
```

Comment:
Is there a simple way to detect if the `solutionLatex` actually contains LaTeX content? Will a latex string always contain `\\`?

If so, it may be best  to check for the existence of latex, and if it isn't there, we don't need to add the `mathquill-embed-latex` class to the corresponding div, so we won't waste time converting plaintext.